staging/src/k8s.io/cloud-provider/controllers/service/controller.go

```diff
@@ -658,6 +660,9 @@ func nodeNames(nodes []*v1.Node) sets.String {
 }
 
 func shouldSyncUpdatedNode(oldNode, newNode *v1.Node) bool {
+	if utilfeature.DefaultFeatureGate.Enabled(features.StableLoadBalancerNodeSet) {
```

Comment:
this should have a comment explaining it 😄 if you are not familiar with this is impossible you can infer what this does :

Reply:
I can add a short comment mentioning: "this is intended to reduce the number of re-configurations for the load balancer node set". But honestly, there's an entire KEP for people to read if they are interested in understanding the "why"?

Reply:
I don't mean to be exhaust, but what is the expected output of this ? something as "only nodes with ... will be updated"

```
respectsPredicates(oldNode, nodeIncludedPredicate) != respectsPredicates(newNode, nodeIncludedPredicate)
```

Reply:
>  but what is the expected output of this ? 

Only Nodes with changes to the label `"node.kubernetes.io/exclude-from-external-load-balancers"` will trigger a load balancer re-sync.

Reply:
Want me to add that as a doc bloc?

Reply:
yeah please, if someone like me has to debug something in this area, that will give me a hint, otherwise I have to do `git blame, read KEP, ...`

```
// only Nodes with changes to the label "node.kubernetes.io/exclude-from-external-load-balancers" will trigger a load balancer re-sync.
```